ansible/roles/iroha-docker/defaults/main.yml

```
@@ -151,9 +154,24 @@ iroha_service_host: False
 iroha_service_account: ''
 #  The number of keys must match the quorum settings
 iroha_service_account_keys: []
+
+## Postgres - rdb database configs
+# iroha_new_database_config: False and iroha_use_rdb: False stand for old Postgres config
+# iroha_new_database_config: True and iroha_use_rdb: False stand for new Postgres config
+# iroha_new_database_config: True and iroha_use_rdb: True stand for rdb config
+# iroha_new_database_config: False and iroha_use_rdb: True is not valid
 iroha_new_database_config: False
 iroha_use_rdb: False
```

Maybe we can join these two variables?
iroha_use_rdb: False - old Postgres config
iroha_use_rdb: True - new Postgres config

I don't see cases when someone wants to use  `iroha_new_database_config: True and iroha_use_rdb: False`. 

```suggestion
iroha_use_rdb: False
iroha_new_database_config: "{{ iroha_use_rdb }}"
```